src/components/fb/CreateFidelityBond.tsx

```diff
@@ -645,34 +645,42 @@ const CreateFidelityBond = ({ otherFidelityBondExists, wallet, walletInfo, onDon
         </PaymentConfirmModal>
       )}
 
-      <div className={styles.header} onClick={() => setIsExpanded(!isExpanded)}>
-        <div className="d-flex justify-content-between align-items-center">
-          <div className={styles.title}>
-            {otherFidelityBondExists
-              ? t('earn.fidelity_bond.title_fidelity_bond_exists')
-              : t('earn.fidelity_bond.title')}
-          </div>
-          <Sprite symbol={isExpanded ? 'caret-up' : 'plus'} width="20" height="20" />
-        </div>
-        <div className={styles.subtitle}>
-          {otherFidelityBondExists ? (
-            <Trans i18nKey="earn.fidelity_bond.subtitle_fidelity_bond_exists">
-              <a
-                onClick={(e) => e.stopPropagation()}
+      {otherFidelityBondExists ? (
+        <div className={styles.containerWhenBondAlreadyExists}>
+          <div className={styles.headerWhenBondAlreadyExists} onClick={() => setIsExpanded(!isExpanded)}>
+            <div className="d-flex justify-content-around align-items-center">
+              <div className={styles.title}>{t('earn.fidelity_bond.title_fidelity_bond_exists')}</div>
+              <Sprite symbol={isExpanded ? 'caret-up' : 'plus'} width="15" height="15" />
+            </div>
+            {/* <div className={styles.subtitle}>
+              <Trans i18nKey="earn.fidelity_bond.subtitle_fidelity_bond_exists">
```

Comment:
Do you have an idea where we can display the message? Maybe after the users actively clicks the button?